src/app/tech/tech.component.html

```diff
@@ -44,15 +33,19 @@ <h3>You may list as many pieces of tech as you want.</h3>
                 </p>
             </div>
         </div>
-        <div class="half unit row">
-            <fieldset>
-                <input type="button" value="Add Tech." (click)="addTechListing()">
-            </fieldset>
+        <div class="one half unit row">
+            <div class="two unit column">
+                <button (click)="addTechListing()">Add Tech.</button>
+            </div>
         </div>
-        <div class="half unit row" *ngFor="let techListing of techListings; let i = index">
-            <fieldset>
+        <fieldset class="half unit persistent row content centred" *ngFor="let techListing of techListings; let i = index">
```

Comment:
When you have multiple items in the list of tech it starts to look cramped. Can we have a margin between each item? Thanks!